### PR TITLE
Bump addon-manager's version to v9.1.0

### PR DESCRIPTION
**Pull request by tosi3k**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
This PR is needed to release #91018. 

I will update the manifests in a separate PR.

**Which issue(s) this PR fixes**:

No issue is opened for that.

**Special notes for your reviewer**:
/sig scalability
/assign mm4tt

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/addons/addon-manager/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## Version 9.0.3 (Wed May 13 2020 Antoni Zawodny <zawodny@google.com>)
+ - Enable overriding the default list of whitelisted resources
```
Comment (wojtek-t) [inline cluster/addons/addon-manager/CHANGELOG.md:2]: This is not a bug fix. So it shouldn't be patch release imho.

I suggest 9.1.0 instead

Comment (tosi3k) [reply cluster/addons/addon-manager/CHANGELOG.md:2]: Done. Thanks for pointing that out.

**Pull request merged**